src/main/java/uk/gov/companieshouse/model/enums/DissolutionStatus.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.model.enums;
+
+public enum DissolutionStatus {
+    PENDING_APPROVAL("pending-approval"),
+    PENDING_PAYMENT("pending-payment"),
+    PAID("paid");
+
+    public final String label;
+
+    DissolutionStatus(String label) {
+        this.label = label;
```

Comment:
I can't see anywhere in the implementation that we access `.label` to access the lowercase value - does Java do this automatically?

Reply:
Yes

Reply:
However, if you want to pass just the enum value, e.g. in a function that needs a string, you will need to invoke .label. I will add a getter to that, after all.

Reply:
As long as the swagger docs display:
```
application_status
enum
[
  "pending-approval",
  "pending-payment",
  "paid"
]
```

then I don't mind. This is a CH API standard so we should never display the uppercase version in the Swagger.